pkg/controller/spire-server/configmap.go

```diff
@@ -142,10 +142,10 @@ func generateServerConfMap(config *v1alpha1.SpireServerSpec) map[string]interfac
 					"organization": []string{config.CASubject.Organization},
 				},
 			},
-			"ca_ttl":                "24h",
+			"ca_ttl":                config.CAValidity,
 			"data_dir":              "/run/spire/data",
-			"default_jwt_svid_ttl":  "1h",
-			"default_x509_svid_ttl": "4h",
+			"default_jwt_svid_ttl":  config.DefaultJWTValidity,
+			"default_x509_svid_ttl": config.DefaultX509Validity,
```

Comment:
Just an idea - Is it necessary to include a validation check to ensure that the value of `default_*_svid_ttl` is less than `ca_ttl`, or would it be sufficient to simply add a note in the RH doc.